src/assets/Enums.ts

```diff
@@ -28,7 +28,8 @@ export enum ERRORS {
     OdometerReadingUnitsMandatory = "\"odometerReadingUnits\" is mandatory",
     VehicleSubclassIsNotAllowed = "\"vehicleSubclass\" is not allowed",
```

Comment:
I think we start to have many enums for the ERRORS, is it worth maybe adding a comment indicate which section it refers to? It could help locating groups of errors rather than going through all of them if we were to visually scan this enum? Thoughts?

Reply:
These last few errors in particular are only used for unit tests. Personally I wouldn't even store them in the Errors enum. It was the decision taken by a previous tech-lead and that's why you can see this behaviour in different services. If you think that it might improve the readability, I can split them.

Reply:
I am not sure yet how this list could grow and how many more error messages we could have. I was thinking about having some comment for now indicating what they are about but if you think it makes sense to separate them, we could do it. There have a been a few instances where it was suggested to move them 'somewhere' to avoid hardcoding them. It is probably a better idea if everyone agrees bering in mind that it should reflect some sort of domain business for consistency to facilitate communication with the business preferably. @m-ujaffer @KHodderDVSA thoughts?

Reply:
Requested changes done. Separated enum values used for testing only.